code/game/objects/items.dm

```diff
@@ -530,3 +539,13 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 
 /obj/item/proc/is_equivalent(obj/item/I)
 	return I == src
+
+/obj/item/Crossed(atom/movable/AM)
+	if(prob(trip_chance) && ishuman(AM))
+		var/mob/living/carbon/human/H = AM
+		on_trip(H)
+
+/obj/item/proc/on_trip(mob/living/carbon/human/H)
+	if(H.slip(src, trip_stun, trip_weaken, trip_tiles, trip_walksafe, trip_any, trip_verb))
+		H.visible_message("<span class='warning'>[H] [trip_verb]s on [src]!</span>")

```

Comment:
I have it as a visible message so that other people can identify what someone tripped on, since otherwise it could be ambiguous.